framework/src/play/libs/Expression.java

```diff
@@ -21,7 +21,7 @@
 
 public class Expression {
 
-  private static final Pattern expression = Pattern.compile("^\\$\\{(.*)\\}$");
+  private static final Pattern expression = Pattern.compile("^\\$\\{(.*)}$");
```

Comment:
Same question here: what's the motivation behind this change?

Or in other words: why removing `\` from the closing `}` but not from the opening `{`?

Reply:
The `\\` is useless there. For the `{` it is needed, for the `}` it is not. These are from IntelliJ's regex hints, and they thus far have always been on point.